gulpfile.babel.js

```diff
@@ -59,7 +67,7 @@ gulp.task('apidoc', ['clean:apidoc'], function(done){
   apidoc( apidocConfig ,done);
 });
 
-gulp.task('run:eslint', () => gulp.src('src/**/*.js')
+gulp.task('run:eslint', () => gulp.src(['src/**/*.js', '!src/__tests__/browser/components/*.js'])
```

Comment:
I reverted! and I was able to test it under Jest env by adding Jest:true in .eslintrc